text/3533-combine-infra-and-release-teams.md

```diff
@@ -0,0 +1,57 @@
+
+With this proposal the Infrastructure team's purview does not grow considerably and remains at a level similar to other existing top-level teams such as a language and compiler.
+
+For the purposes of actual decision making, the Release subteam retains all decision-making power with regard to release related issues (i.e., this proposal does not change who makes any particular decision and is purely a change in Council representation). This may change over time should the Infra team choose to structure itself in a different way.
+
+# Practicalities
+
+Once this proposal is accepted, the Release team will move to be a subteam of Infrastructure. The Infrastructure team does not change its top-level form.
+
+The Infrastructure team's Council representative would continue to serve on the Council while the Release representative would immediately stop counting as a representative for all purposes.[^plan]
+
+# Alternatives
+
+## Combining Infra and Release into a new team
+
+Instead of merging the Release team into Infrastructure, a new team is formed consisting of Infrastructure and Release as the two subteams.
+
+This option is much more complex logistically speaking for the following reasons:
+* It is unclear what this team's purview would actually be (i.e., we end up in the same situation as the status quo - one team with a larger purview than the other - but now within this new team)
+* Some process would need to be created to decide who the team leads would be.
+
+It is not clear that there are any benefits to this alternative that are worth the cost of the above complexities.
+
+# Future Possibilities
+
+## Merge part of crates.io into Infrastructure
+
+Crates.io is arguably the other team next to Release with the smallest purview compared to other teams. We may want to merge at least part of the team (i.e., the part responsible for running the crates.io infra) into Infrastructure.
+
+This is not included in this proposal, because there are more open questions in this case than in the case of the Release team (e.g., where would the other part of the crates.io team - in charge of crates.io policy - go?).
+
+# Open questions
+
+* Should the team still be called *Infrastructure*?
+* What happens to wg-triage? Should this be moved out of Release as a working group of Infrastructure? Should it instead move to another team (perhaps compiler)?
```

Comment:
We should probably settle these questions as part of this RFC, given that there's no real "implementation" follow-up? My personal proposal would be:

* Keep the top-level team name. Release work is in many ways still "infra" work (IMO) and renaming is not worth the hassle.
* Move wg-triage out into launching pad, pending a separate RFC or other proposal on the best home for it. In practice there's very little check-in or other relationship today with t-release <> wg-triage under it, so I think this move won't make any real practical delta.

cc @Dylan-DPC as the [current lead of wg-triage](https://github.com/rust-lang/team/blob/master/teams/wg-triage.toml)

Reply:
I'm totally fine with moving it to launch pad. As a working group we are mostly independent and cater to all the teams so it's the best option for a neutral ground. If an RFC is needed, i'll get to writing it later this week

Reply:
I've noted that wg-triage will be moved to Launching Pad.